mito-ai/src/Extensions/AiChat/SignUpForm.tsx

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) Saga Inc.
+ * Distributed under the terms of the GNU Affero General Public License v3.0 License.
+ */
+
+import React, { useState } from 'react';
+import { setUserKey } from '../../restAPI/RestAPI';
+import '../../../style/SignUpForm.css';
+
+
+interface SignUpFormProps {
+    onSignUpSuccess?: () => void;
+}
+
+const SignUpForm: React.FC<SignUpFormProps> = ({ onSignUpSuccess }) => {
+    const [email, setEmail] = useState<string>('');
+
+    const handleSubmit = async (e: React.FormEvent): Promise<void> => {
+        e.preventDefault();
+        try {
+            await setUserKey('user_email', email);
+            onSignUpSuccess?.();
+        } catch (error) {
+            console.error('Failed to set user email:', error);
+        }
+    };
+
+    return (
+        <div className="signup-form-container">
+            <div className="signup-form-message" data-testid="signup-form-message">
+                Get started with Mito, no credit card required, just an email.
+            </div>
+            <form onSubmit={handleSubmit} style={{ marginTop: '15px' }}>
+                <input
+                    type="email"
+                    value={email}
+                    onChange={(e) => setEmail(e.target.value)}
+                    placeholder="Your work email"
+                    className="signup-form-input"
+                    required
+                />
+                <button
+                    type="submit"
+                    className="button-base button-purple signup-form-button"
+                >
+                    Sign Up
+                </button>
+            </form>
```

Comment:
I think we could make this pop a little bit more in dark mode. The disabled colors are less clear and the signup form itself is a little not standing out to me. 

What if we: 
1. Make the Sign Up Button a darker purple color?
2. Make the "Sign Up for Mito" title text a stronger color? In dark mode it could be white. In light mode it could be black. There must be a Jupyter color variable we can use for that. 

<img width="387" height="908" alt="Image" src="https://github.com/user-attachments/assets/3cf743d4-b8bd-4a05-9dd8-a11f3490d544" />